tests/data/test_preprocess_cord.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for making datasets for contradictory-claims."""
+
+# -*- coding: utf-8 -*-
+
+import os
+import unittest
+from datetime import datetime
+
+# from contradictory_claims.data.preprocess_cord import filter_metadata_for_covid19
+
+from .constants import metadata_path, pub_date_cutoff, virus_lex_path
+
+
+class TestPreprocessCord(unittest.TestCase):
+    """Tests for preprocessing CORD-19."""
+
+    def test_find_files(self):
+        """Test that input files are found properly."""
+        self.assertTrue(os.path.isfile(metadata_path),
+                        "Metadata.csv not found at {}".format(metadata_path))
+
+        self.assertTrue(os.path.isfile(virus_lex_path),
+                        "Virus lexicon not found at {}".format(virus_lex_path))
+
+    def test_pub_date_cutoff(self):
```

Comment:
does this test need an assertion?

Reply:
Not sure...assertIsInstance and assertRaises both fail for me (I've included the code I tried in comments now)..

Reply:
hmm.